google/cloud/ndb/_transaction.py

```diff
@@ -58,9 +175,9 @@ def transaction(
         xg (bool): Enable cross-group transactions. This argument is included
             for backwards compatibility reasons and is ignored. All Datastore
             transactions are cross-group, up to 25 entity groups, all the time.
-        propagation (Any): Deprecated, will raise `NotImplementedError` if
-            passed. Transaction propagation was a feature of the old Datastore
-            RPC library and is no longer available.
+        propagation (int): Transaction propagation was a feature of the old
```

Comment:
The reader doesn't need to know the history. They do need to know what it is.

Reply:
b8f26ff